clients/hadoop-to-venice-bridge/src/main/java/com/linkedin/venice/hadoop/input/kafka/ttl/TTLResolutionPolicy.java

```diff
@@ -0,0 +1,60 @@
+package com.linkedin.venice.hadoop.input.kafka.ttl;
+
+import com.linkedin.venice.exceptions.VeniceException;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * The policy controls the TTL behavior regarding how batch writes are treated.
+ * As of today (09/15/2022), only real-time write can be TTLed/supported.
```

Comment:
Commit history serves as a good timeline. Can we remove the date from here?

Reply:
Ack.